src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+body,
+header,
+a,
+img,
+div,
+nav,
+ul,
+li {
+  margin: 0;
+}
```

Comment:
it's redundant, why we need to reset margins for `header` or `a` for example? not all this tags have margins by default